verification/curator-service/ui/src/components/common-form-fields/Source.tsx

```diff
@@ -42,17 +51,62 @@ interface ListSourcesResponse {
 }
 
 interface SourceAutocompleteProps {
-    initialValue: string;
+    initialValue?: CaseReference;
 }
 
+interface AdditionalSource {
+    sourceUrl: string;
+}
+
+interface CaseReference extends CaseRef {
```

Comment:
optional nit: it would be nice if the different between CaseReference and CaseRef was more clear by name. Maybe something like CaseReferenceInput and CaseReference or CaseReferenceUI and CaseReferenceSchema?